zuul.d/projects.yaml

```diff
@@ -4,8 +4,8 @@
     github-check:
       jobs:
         - install-yamls-content-provider
-        - install-yamls-crc-podified-edpm-baremetal: &content_provider
+        - install-yamls-crc-podified-edpm-deployment:
             dependencies:
               - install-yamls-content-provider
-        - install-yamls-crc-podified-galera-deployment: *content_provider
-        - install-yamls-crc-podified-edpm-deployment: *content_provider
+            vars:
+              cifmw_edpm_deploy_run_validation: true
```

Comment:
+100
but I think we need https://github.com/openstack-k8s-operators/install_yamls/pull/512 to make it stable. (Feel free to squash that into this PR)